pkg/api/domain.go

```diff
@@ -12,6 +12,7 @@ const (
 	ServiceDomainAPPCI = "apps.ci.l2s4.p1.openshiftapps.com"
 
 	ServiceDomainAPPCIRegistry = "registry.ci.openshift.org"
+	ServiceAPPCIRegistry       = "registry.svc.ci.openshift.org"
```

Comment:
What is this? We shouldn't need to point any traffic to this DN

Reply:
It comes from https://github.com/openshift/release/pull/20469/files#diff-6f809450f5216bc90d0c08b723c9fe080da1358283bbf47c42f05bfc589c49fdR2308 in the example PR that hongkai gave me.

Reply:
@hongkailiu ? What is this?